### PR TITLE
Second_commit

### PR DESCRIPTION
**Pull request by MartiCobos99**

### CODE REVIEW

File: exercise_03/generateRandomNumberInRange.js
Comment (tictools) [inline exercise_03/generateRandomNumberInRange.js:1]: FIX: missing function signature

File: exercise_04/isGreaterThan.js
Comment (tictools) [inline exercise_04/isGreaterThan.js:1]: FIX: missing function signature

File: exercise_05/isAdditionGreaterThanFifty.js
Comment (tictools) [inline exercise_05/isAdditionGreaterThanFifty.js:1]: FIX: missing function signature

File: exercise_06/checkCollectionHasElements.js [hunk 1]
```diff
@@ -3,3 +3,7 @@
  * input: array (collection)
  * output: boolean
  */
+
+export function checkCollectionHasElements(collection) {
+  return Array.isArray(collection) && collection.length > 0;
```
Comment (tictools) [inline exercise_06/checkCollectionHasElements.js]: Be careful!

`Array.isArray(collection)` will return `false` if provided collection is, for example, a number. Even result will be a boolean, this should be treated apart, as validation.

File: exercise_07/index.js [hunk 2]
```diff
@@ -1 +1,13 @@
...
+const stringValue = "Hello";
+const stringType = "string";
+const resultString = isTypeOf(stringValue, stringType);
+console.log(`Is "${stringValue}" of type ${stringType}? ${resultString}`);
+
+const arrayValue = [1, 2, 3];
+const arrayType = "array";
+const resultArray = isTypeOf(arrayValue, arrayType);
+console.log(
+  `Is ${JSON.stringify(arrayValue)} of type ${arrayType}? ${resultArray}`,
```
Comment (tictools) [inline exercise_07/index.js:12]: Nice use of `JSON.stringfy`. I assume you really know what it does and when you can use it. 😉

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by nilete22 — APPROVED** [review]
approve

**Review by nilete22 — APPROVED** [review]

**Review by tictools — APPROVED** [review]

**Pull request merged**